### PR TITLE
Aws rds snapshot remove publicaccess

### PR DESCRIPTION
**Pull request by sreedevikr**
This commit includes the changes for the custom remediation job for restricting public access for the RDS Snapshot. Could you also please share the details about the rule id that needs to be updated. We have currently not updated the rule id .

### CODE REVIEW

**Review by Manmohan-Tyagi — APPROVED** [review]

File: remediation_worker/jobs/aws_rds_snapshot_remove_publicaccess/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Disable public access to RDS Snapshots
+
+This job removes public access from RDS snapshots and makes the snapshots private. 
+
+## Getting Started
+
+##### Rule ID:
+5c8c26467a550e1fb6560c48
```
Comment (kshrutik) [inline remediation_worker/jobs/aws_rds_snapshot_remove_publicaccess/README.md:8]: This is the Rule Id for publicly accessible snapshot - 5c8c26487a550e1fb6560c4a
The one which you have used is for publicly accessible rds DB instance

**Review by kshrutik — CHANGES_REQUESTED** [review]

**Comment by kshrutik** [comment]
Please add the unit test for this remediation job.

File: remediation_worker/jobs/aws_rds_snapshot_remove_publicaccess/README.md (hunk 1 above)
Comment (sreedevikr) [reply remediation_worker/jobs/aws_rds_snapshot_remove_publicaccess/README.md]: Thanks @kshrutik  for the review comments. I have corrected the ruleid and also added the unit tests. Could you please review.

**Comment by kshrutik** [comment]
Also, can you please update the main readme file and the tox file? Run the tox file and verify if the unit tests are passing.
Refer this [PR](https://github.com/vmware-samples/secure-state-remediation-jobs/pull/97)

**Comment by sreedevikr** [comment]
Thanks @kshrutik for the review comments. I have updated the main readme and the tox file , also the ruleid in the remediation job readme was not updated correctly in the previous commit. I have rectified that too. Thanks.

**Review by kshrutik — APPROVED** [review]

**Review by kshrutik — APPROVED** [review]

**Pull request merged**